src/imap_db/model.py

```diff
@@ -26,7 +26,7 @@ class File(Base):
     version: Mapped[int] = mapped_column(Integer())
     hash: Mapped[str] = mapped_column(String(64))
     size: Mapped[int] = mapped_column(Integer())
-    content_date: Mapped[datetime] = mapped_column(DateTime())
+    content_date: Mapped[datetime] = mapped_column(DateTime(), nullable=True)
```

Comment:
Sorry - if you want to change the database schema you need to do a database migration using alembic. The prod database needs to have to schema changed to make that field nullable. The CLI tools should make one automatically but you need to ask for it. Let me know if you want me to show you this,